lib/editing/editor.dart

```diff
@@ -7,6 +7,7 @@ library editor;
 import 'dart:async';
 import 'dart:html' as html;
 import 'dart:math';
+import 'package:dart_pad/dartservices_client/v1.dart';
```

Comment:
We want to avoid having the editor library depend on the services one. The editor library should be a bit more abstract than that. I think we want an implementation of a `SourceEdit` like class to live in this library. It would be similar to the services `Edit` class. People using an `Editor` instead that had made a services call would need to convert between the `Edit` class and the `SourceEdit` one; similar to what we do for code completions.